SKILL.tsv

```diff
@@ -2443,24 +2443,24 @@ SKILL_20150317_002442	$Rush: Stun
 SKILL_20150317_002443	{memo X}Enemies pushed away by [Rush] are inflicted with [Stun] for 2 seconds at a 5% chance per attribute level.
 SKILL_20150317_002444	$Impaler: Double Damage
 SKILL_20150317_002445	The original monster hit by[Impaler] receives double damage.
-SKILL_20150317_002446	Dragging Death: Control ally trap
-SKILL_20150317_002447	[Dragging Death] can drag traps and statues installed by allies.
-SKILL_20150317_002448	Stead Charge: Additional damage against Large monsters
+SKILL_20150317_002446	$Dragging Death: Ally Trap Control
+SKILL_20150317_002447	$Allies can drag prepared traps and statues of [Dragging Death].
+SKILL_20150317_002448	$Steed Charge: Large-type Additional Damage
```

Comment:
"Additonal damage to Large-type"?

Saying it this way sounds like the damage is Large-type.